remotespark/livyclientlib/livyclient.py

```diff
@@ -8,26 +8,19 @@ class LivyClient(object):
     """Spark client for Livy endpoint"""
     logger = Log()
 
-    def __init__(self, spark_session, pyspark_session):
-        self._spark_session = spark_session
-        self._pyspark_session = pyspark_session
+    def __init__(self, session):
+        self._session = session
 
-    def execute_scala(self, commands):
-        self._spark_session.wait_for_state("idle")
-        return self._spark_session.execute(commands)
+    def execute(self, commands):
+        self._session.wait_for_state("idle")
+        return self._session.execute(commands)
 
-    def execute_scala_sql(self, command):
-        self._spark_session.create_sql_context()
-        return self.execute_scala('sqlContext.sql("' + command + '").collect()')
+    def execute_sql(self, command):
+        self._session.create_sql_context()
+        return self.execute('sqlContext.sql("' + command + '").collect()')
 
-    def execute_pyspark(self, commands):
-        self._pyspark_session.wait_for_state("idle")
-        return self._pyspark_session.execute(commands)
+    def close_session(self):
+        self._session.delete()
 
-    def execute_pyspark_sql(self, command):
-        self._pyspark_session.create_sql_context()
-        return self.execute_pyspark("sqlContext.sql('" + command + "').collect()")
-
-    def close_sessions(self):
-        self._spark_session.delete()
-        self._pyspark_session.delete()
+    def language(self):
```

Comment:
That's what happens with late night coding :) I'll sneak that in the next change.